src/backup.rs

```diff
@@ -46,15 +46,23 @@ impl BackupManager {
             .map_to_permanent_failure("Failed to perform backup of local db")
     }
 
-    pub async fn recover(&self) -> Result<()> {
+    pub async fn recover(&self, external_recovery_is_allowed: bool) -> Result<()> {
```

Comment:
Why not just `allow_external_recovery`?

Reply:
Applied in [69bd723](https://github.com/getlipa/lipa-lightning-lib/pull/1283/commits/69bd72318f569863776ccb56791c5a9c009528aa)